### PR TITLE
Save state of last 5 locations to localStorage 

### PR DESCRIPTION
**Pull request by arinaldi**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
-export function receiveLocation(location, json) {
+export function receiveLocation(newLocation) {
   //dispatching an action to the store
   return {
     type: 'RECEIVE_LOCATION',
-    date: json.datetime,
-    name: location,
-    aqi: json.breezometer_aqi,
-    description: json.breezometer_description
+    date: newLocation.date,
```
Comment (cjessett) [inline src/actions/index.js]: destructure these keys from `newLocation` like `const { date, name, aqi, ...etc } = newLocation`

Comment (cjessett) [reply src/actions/index.js]: then use object assignment shorthand like `return { date, name, aqi, ...etc }`

File: src/actions/index.js [hunk 2]
```diff
@@ -1,25 +1,51 @@
...
+    name: newLocation.name,
+    aqi: newLocation.aqi,
+    description: newLocation.description,
+    color: newLocation.color
   };
 };
 
 export const fetchLocation = location => {
```
Comment (cjessett) [inline src/actions/index.js]: you should be consistent with how you define functions, above you use `function` syntax whereas here you are assigning an anonymous function to a `const`

**Review by jorions — APPROVED** [review]
LGTM 5/5 would CR again

File: src/utilities/index.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    localStorage.removeItem("test");
+    return true;
+  } catch (error) {
+    return false;
+  }
+}
+
+export function getHistory() {
+  const item = localStorage.getItem("airTony");
+  const arrayOfObjects = JSON.parse(item);
```
Comment (cjessett) [inline src/utilities/index.js]: no need to name this, just return the result of parsing

File: src/utilities/index.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+  if (currentHistory.length === 5) newHistory.pop();
+  newHistory.unshift(newLocation);
+  
+  localStorage.setItem("airTony", JSON.stringify(newHistory));
+}
+
+export function formatDate(isoDate) {
+  const date = new Date(isoDate);
+  const newDate = {
```
Comment (cjessett) [inline src/utilities/index.js]: same here just return the thing, don't name it

**Review by cjessett — CHANGES_REQUESTED** [review]

File: src/utilities/index.js (hunk 3 above)
Comment (arinaldi) [reply src/utilities/index.js]: Done.

File: src/utilities/index.js (hunk 4 above)
Comment (arinaldi) [reply src/utilities/index.js]: Done.

File: src/api/index.js [hunk 5]
```diff
@@ -17,3 +17,15 @@ export function breezoMeter(coords) {
     .then(res => res.json())
     .catch(err => {});
 }
+
+// TODO: catch for bad response
```
Comment (ghaynes1292) [inline src/api/index.js]: Lets remove these comments and just add it into the PR for error handling

**Review by ghaynes1292 — CHANGES_REQUESTED** [review]

File: src/api/index.js (hunk 5 above)
Comment (arinaldi) [reply src/api/index.js]: Done.

File: src/actions/index.js (hunk 1 above)
Comment (arinaldi) [reply src/actions/index.js]: Done.

File: src/actions/index.js (hunk 2 above)
Comment (arinaldi) [reply src/actions/index.js]: Done.

**Review by ghaynes1292 — APPROVED** [review]

**Pull request merged**